.drone.star

```diff
@@ -46,7 +46,7 @@ config = {
     "phpstan": True,
     "phan": True,
     "php-unit": True,
-    "ocisBranches":["master", "stable"]
+    "ocisBranches": ["master", "stable"],
```

Comment:
add starlark check pipeline. it will handle the formatting

Reply:
in next PR @saw-jan 😄